### DISCUSSION TITLE
Rearrange screens

### DISCUSSION THREAD

**Discussion by louisepujante** [responsively-org/responsively-app#648] [Q&A] [answered]
Hi, I'd like to ask if there's any way to rearrange the screens? so I can view them in order of larger to smaller screens?

Thank you. :)

**Comment by manojVivek** [comment, marked as answer]
Hi @louisepujante, You can do that in the `device manager` by dragging the devices to your needed order. 

![2021-12-07 11-59-56 2021-12-07 12_05_01](https://user-images.githubusercontent.com/1283424/144978746-fa9861db-ab60-445b-aca1-dd042e45e369.gif)

**Reply by louisepujante** [reply]
Hi @manojVivek, Thank you! That's very helpful. :)
